.agent/workflows/new-milestone.md

```
@@ -0,0 +1,156 @@
+# /new-milestone Workflow
+
+<objective>
+Define a new milestone with goal, phases, and success criteria.
+</objective>
+
+<process>
+
+## 1. Validate SPEC Exists
+
+**PowerShell:**
+```powershell
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)